libs/filabridge/include/private/filament/EngineEnums.h

```diff
@@ -71,6 +72,23 @@ enum class ReservedSpecializationConstants : uint8_t {
     CONFIG_STEREO_EYE_COUNT = 8, // don't change (hardcoded in ShaderCompilerService.cpp)
 };
 
+
+using PushConstantType = backend::ConstantType;
+
+// Note that the following enum/arrays should be ordered so that the ids correspond to indices in
+// the two vectors.
+enum class PushConstantIds {
+    MORPHING_BUFFER_OFFSET = 0,
+};
+const utils::FixedCapacityVector<char const*> PUSH_CONSTANT_NAMES = {
+    "morphingBufferOffset",
+};
+const utils::FixedCapacityVector<PushConstantType> PUSH_CONSTANT_TYPES = {
+    PushConstantType::INT,
+};
+
+constexpr char const* PUSH_CONSTANT_STRUCT_VAR_NAME = "pushConstants";
+
```

Comment:
These should not appear in EngineEnums.h because filament itself will never use them. This should be hidden in some other private header. It's only needed right now because this info is not stored into the material, but it should be. You can add a header next to EngineEnums.h for now (same location).

Reply:
So the "reserved" spec consts are also in this file.  Should they move as well?

Reply:
I guess my point is that this should be written into the material file as a Chunk, if we did that, we wouldn't need these constants in a header shared between filamat and filament (i.e. a filabridge header).

I would prefer that you implement that. But, if not, the next best thing is to put the constants in their own header to make it more clear that they shouldn't be part of EngineEnums.h.

I think I agree the spec constant should work like that as well.

Reply:
I see.  I can put these and push constant into their own headers in this PR, and do a follow-up to put them in Chunks.

Reply:
Actually, looking at the code, it looks like the reserved spec constants are different in that they have default values that need to be set from the filament side (so semantics do have to be shared between filamat and filament). 

I'll update this PR to put just the push constants in a Chunk.

Reply:
done